Source/AwsCommonRuntimeKit/sdkutils/FileBasedConfigSectionType.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import AwsCSdkUtils
+
+/// Type of section in a config file
+public enum FileBasedConfigSectionType {
+    case profile
+    case ssoSession
+}
+
+extension FileBasedConfigSectionType {
```

Comment:
Can we rename this to the following, to be consistent with the other FileBasedConfiguration types? Additionally lets update the file name to match `FileBaseConfigurationSectionType.swift`

```swift
extension FileBasedConfiguration {
    public enum SectionType { 
    // ...
}
```